README.md

```diff
@@ -125,5 +143,6 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
```

Comment:
The schema is the structure of a database. I.e. it defines what tables and columns in a database